app/views/configure/integration.phtml

```diff
@@ -13,7 +13,7 @@
 	<form method="post" action="<?= _url('configure', 'integration') ?>" class="draggableList">
 		<input type="hidden" name="_csrf" value="<?= FreshRSS_Auth::csrfToken() ?>" />
 		<template id="simple-share">
-			<formgroup class="group-share dragbox">
+			<fieldset class="group-share dragbox">
```

Comment:
I apologize very much that I added the no-HTML tag `<formgroup>`  here by accident in the past. 🤣